pkg/cmd/install.go

```diff
@@ -108,16 +108,12 @@ func newCmdInstall(rootCmdOptions *RootCmdOptions) (*cobra.Command, *installCmdO
 	cmd.Flags().String("operator-image-pull-policy", "", "Set the operator ImagePullPolicy used for the operator deployment")
 	cmd.Flags().String("build-strategy", "", "Set the build strategy")
 	cmd.Flags().String("build-publish-strategy", "", "Set the build publish strategy")
+	cmd.Flags().StringArray("build-publish-strategy-option", nil, "Add a build publish strategy option, as <name=value>")
 	cmd.Flags().String("build-timeout", "", "Set how long the build process can last")
 	cmd.Flags().String("trait-profile", "", "The profile to use for traits")
 
 	// Kaniko
-	cmd.Flags().Bool("kaniko-build-cache", false, "To enable or disable the Kaniko cache")
-	cmd.Flags().String("kaniko-executor-image", "", "The docker image of the Kaniko executor")
-	cmd.Flags().String("kaniko-warmer-image", "", "The docker image of the Kaniko warmer")
-
-	// Buildah
-	cmd.Flags().String("buildah-image", "", "The docker image to use for Buildah")
```

Comment:
Cannot recall if this one was present in 1.10 release already. If so, we should not remove but add a deprecation notice.

Reply:
No, it has never been released. Same for `kaniko-warmer-image` and `kaniko-executor-image`. They have been added by https://github.com/apache/camel-k/pull/3558